source/Projectile.cpp

```diff
@@ -71,18 +72,20 @@ Projectile::Projectile(const Projectile &parent, const Point &offset, const Angl
 	cachedTarget = TargetPtr().get();
 	double inaccuracy = weapon->Inaccuracy();
 	if(inaccuracy)
-	{
 		this->angle += Angle::Random(inaccuracy) - Angle::Random(inaccuracy);
-		if(!parent.weapon->Acceleration())
-		{
-			// Move in this new direction at the same velocity.
-			// To maintain the sign of the velocity, Point::Length can’t be used.
-			Point referenceVector = parent.angle.Unit();
-			double parentVelocity = referenceVector.Dot(parent.velocity);
```

Comment:
I'm guessing that the issue with the old system is that `parent.velocity` includes the velocity of the ship that fired the projectile, whereas the now used `velocity -= parent.angle.Unit() * parent.speed;` rewinds us to the firing ship's velocity. That is to say, the behavior seen in #6695 is because submunitions are applying the inaccuracy to the firing ship's velocity when we don't actually want to touch that, hence why this PR functions better.

Reply:
Yep FerociousFiend said something like that.  Naively added speeds and naively added angles combine together improperly I think it was.

Reply:
Although with that said, an alternative to the approach this PR takes would be to pass along a Point vector that only accounts for the velocity of the projectile and use that here.
```
In the first constructor:
velocity = firing ship velocity
projVel = angle * projectile velocity
velocity += projVel

In the submunition constructor:
projVel = parent projVel
projVel += angle * projectile velocity
velocity += projVel

In the Move function:
Change projVel the same way that velocity is changes.
This would mean that accelerating parent projectiles could apply inaccuracy to submunitions.
```